docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -463,11 +473,12 @@ code: |
   add_includes = True
 ---
 code: |
-  short_title_block = interview.blocks.appendObject()
-  short_title_block.template_key = 'code'
-  short_title_block.data = {
-    "lines": ['interview_short_title = "' + escape_quotes(interview_intro_prompt) + '"']
-    }
+  if defined('interview_intro_prompt'):    
+    short_title_block = interview.blocks.appendObject()
+    short_title_block.template_key = 'code'
+    short_title_block.data = {
+      "lines": ['interview_short_title = "' + escape_quotes(interview_intro_prompt) + '"']
+      }
```

Comment:
```suggestion
  short_title_block = interview.blocks.appendObject()
  short_title_block.template_key = 'code'
  short_title_block.data = {
    "lines": ['interview_short_title = "' + escape_quotes(interview_intro_prompt) + '"']
    }
```
Not needed as we're keeping interview_intro_prompt for now. People can still add their own branding page--there is a stock one that comes with stock assembly line.